smalltalksrc/Slang/CSlangPrettyPrinter.class.st

```diff
@@ -107,6 +107,14 @@ CSlangPrettyPrinter >> visitDeclarator: aDeclarator [
 	aDeclarator declarator acceptVisitor: self
 ]
 
+{ #category : #generated }
+CSlangPrettyPrinter >> visitGotoStatement: aGoToStatement [
+
+	stream nextPutAll: 'goto ';
+			 nextPutAll: aGoToStatement label name.
```

Comment:
Maybe you can reformat this one :)
Also, no need to put a dot at the end of the last statement of a method.